compiler/src/dotty/tools/dotc/core/tasty/TreeUnpickler.scala

```diff
@@ -97,6 +99,14 @@ class TreeUnpickler(reader: TastyReader,
   /** Was unpickled class compiled with capture checks? */
   private var withCaptureChecks: Boolean = false
 
+  private val unpicklingScala2Library =
+    attributeUnpicklerOpt.exists(_.attributes.scala2StandardLibrary)
+
+  /** This dependency was compiled with explicit nulls enabled */
+  // TODO Use this to tag the symbols of this dependency as compiled with explicit nulls (see use of unpicklingScala2Library).
```

Comment:
Should we tag the top-level class with an annotation?

Reply:
I will need help from someone familiar with explicit nulls to finish this side of the implementation. We can do that in another PR. I could also remove this attribute temporarily until we have the full implementation.

Reply:
What matters for now is that the information gets stored. If it's not *used* by dotc at the moment, then we don't care.